src/test/java/karate/balance/tests.feature

```diff
@@ -17,6 +18,147 @@ Feature: Balance Tests
     }
     """
 
+    * def getToday =
+    """
+    function() {
+      var dt = new Date();
+      var month = dt.getMonth() + 1;
+      var formattedMonth = ("0" + month).slice(-2);
+      var formattedDay = ("0" + dt.getDate()).slice(-2);
+
+      return dt.getFullYear() + "-" + formattedMonth + "-" + formattedDay;
+    }
+    """
+
+    * def defaultStockPostBody =
+    """
+    {
+      name: "Test Product",
+      unitType: "PIECE",
+      quantity: 100.0,
+      pricePerUnit: 2,
+      sustainablyProduced: true,
+      originCategory: "UNKNOWN",
+      producer: "producer",
+      supplier: "supplier",
+      stockStatus: "INSTOCK",
+      vat: 0.19
+    }
+    """
+
+  Scenario: GET /balance/:userId/history works for user with almost empty balance history
```

Comment:
Isn't his users balanceHistory not "almost" empty but definitely empty?

Reply:
Yes, corrected.